src/Core/Form/IdentifiableObject/DataHandler/ManufacturerFormDataHandler.php

```diff
@@ -43,17 +43,17 @@ final class ManufacturerFormDataHandler implements FormDataHandlerInterface
      */
     private $bus;
     /**
-     * @var ImageUploaderInterface
```

Comment:
Argh 😞 I see why you do this, but it would be better to rely on Interface than implementation to allow people to extend PrestaShop

The best would be to improve ImageUploaderInterface but
- it would be a BC break
- does it make sense to have `ImageUploaderInterface:: checkImageIsAllowedForUpload()` ? does it make sense? it's not the responsibility of an ImageUploader

So I 👍 the chosen solution although I wished we could keep an interface in ManufacturerFormDataHandler

Reply:
Good point. We have for example `CategoryMenuThumbnailUploader` where we don't use `AbstractImageUploader` so there's no validation based on `UploadedFile`, I think changing this interface will require a little restructuration of upload in these places.

In the end, this is a bug fix so maybe we could afford to have BC Break here.